pkg/daemon/daemon.go

```diff
@@ -129,23 +131,25 @@ func New(
 	client snclientset.Interface,
 	kubeClient *kubernetes.Clientset,
 	mcClient *mcclientset.Clientset,
+	isHypershift bool,
```

Comment:
that's correct! and sure

Reply:
well actually @bn222: in order to make that a "dynamic function" we need to add a new clientSet into the daemonset and more state to track if it has been initialized. It can be done but it will result in more state, and complexity to support the same use case.